inlong-sort-standalone/sort-standalone-source/src/main/java/org/apache/inlong/sort/standalone/sink/elasticsearch/EsSinkContext.java

```diff
@@ -565,21 +550,25 @@ public void setUseIndexId(boolean isUseIndexId) {
     }
 
     /**
-     * get indexRequestHandler
+     * create indexRequestHandler
      * 
      * @return the indexRequestHandler
      */
-    public IEvent2IndexRequestHandler getIndexRequestHandler() {
-        return indexRequestHandler;
-    }
-
-    /**
-     * set indexRequestHandler
-     * 
-     * @param indexRequestHandler the indexRequestHandler to set
-     */
-    public void setIndexRequestHandler(IEvent2IndexRequestHandler indexRequestHandler) {
-        this.indexRequestHandler = indexRequestHandler;
+    public IEvent2IndexRequestHandler createIndexRequestHandler() {
+        // IEvent2IndexRequestHandler
+        String indexRequestHandlerClass = CommonPropertiesHolder.getString(KEY_EVENT_INDEXREQUEST_HANDLER,
+                DefaultEvent2IndexRequestHandler.class.getName());
+        try {
+            Class<?> handlerClass = ClassUtils.getClass(indexRequestHandlerClass);
+            Object handlerObject = handlerClass.getDeclaredConstructor().newInstance();
+            if (handlerObject instanceof IEvent2IndexRequestHandler) {
+                IEvent2IndexRequestHandler handler = (IEvent2IndexRequestHandler) handlerObject;
+                return handler;
+            }
+        } catch (Throwable t) {
+            LOG.error("Fail to init IEvent2IndexRequestHandler,handlerClass:{},error:{}",
```

Comment:
In the error log, it is suggested to print all exception stacks.

Reply:
ok, print all exception stacks.